tests/functional_small/type_traits/traits_eigen.cc

```diff
@@ -7,15 +7,18 @@ namespace pressio { namespace traits { namespace test {
 template <typename T, int rank, bool is_dynamic>
```

Comment:
if is_dynamic not used , can you remove it?

Reply:
thanks for catching this - fixed on a4f80c147136f15c9f01d141523877a5dd68e153

Reply:
thanks!